fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -109,7 +114,7 @@ const createElection = (
  */
 
 const CreateElection = ({ route }: any) => {
-  const styles = route.params;
+  const routeStyles = route.params;
```

Comment:
isn't it a bit weird here that we get the styles from the route params? 🤔 
why is that ?

Reply:
It is very weird indeed! I have not yet figured out where these styles come from.. The same code is in CreateMeeting and CreateRollCall and the styles come from here:

https://github.com/dedis/popstellar/blob/master/fe1-web/src/features/events/screens/CreateEvent.tsx#L16-L27

I will look into changing this in a separate PR :)